app/controllers/application_controller.rb

```diff
@@ -1,5 +1,17 @@
 class ApplicationController < ActionController::Base
+  before_filter :authenticate
+
   include Slimmer::GovukComponents
 
   protect_from_forgery with: :exception
+
+  private
+
+  def authenticate
+    return unless Rails.env.production?
```

Comment:
It's a wrapper around `ENV["RAILS_ENV]"` http://api.rubyonrails.org/classes/Rails.html#method-c-env